i18n/fr/docusaurus-plugin-content-docs/version-22.10/connect/openid.md

```diff
@@ -12,21 +12,19 @@ tout autre IdP compatible avec le flux d'autorisation via un code (Authorization
 
 ## Configurer l'authentification OpenID Connect
 
-L'authentification se paramètre à la page **Administration > Authentification > OpenId Connect Configuration** :
-
-![image](../assets/administration/oidc-configuration.png)
+L'authentification se paramètre à la page **Administration > Authentification > OpenId Connect Configuration**.
```

Comment:
In french, the name of the page is Administration > Authentification > Configuration OpenId Connect